src/lucky/action_callbacks.cr

```diff
@@ -127,10 +127,10 @@ module Lucky::ActionCallbacks
       #   end
 
       if callback_result.is_a?(Lucky::Response)
-        Lucky::ActionCallbacks.log_stopped_callback(context,"{{ callback_method.id }}")
+        Lucky::ActionCallbacks.log_stopped_callback("{{ callback_method.id }}")
```

Comment:
Yeah, I see what you mean. This should be `Lucky::ActionPipes.log_stopped_pipe`, or we go full on callback everywhere. I do like "pipe", but maybe we don't fight it since typing `callback` just seems to be what you want to do first?